dist/s_all

```diff
@@ -45,95 +62,81 @@ echo "Updating files that include the package version" &&
 errchk()
 {
     if ! `test -s $2`; then
-        return
+        return 0
     fi
 
-    echo "####################### MESSAGE ############################"
-    echo "s_all run of: \"$1\" resulted in:"
-    sed -e 's/^/    /' $2
-    echo "#######################"
+    echo '#########' s_all run of: "$1" resulted in:$'\n'"$(cat $2)"$'\n''#########'
 
-    # If the test was skipped or marked warning only, ignore the failure.
-    if [ ! `grep "$1.*skipped" $2 > /dev/null` ] && [ ! "$3" = "--warning-only" ] ; then
+    if grep -q "$1.*skipped" $2 || [[ "$3" = "--warning-only" ]]; then
+        : # If the test was skipped or marked warning only, ignore the failure.
+    else
         errfound=1;
+        echo "$1" >> "${t_pfx}_errfound"
     fi
 
     rm -f $2
+    return $errfound
```

Comment:
I don't think this does anything. The return val from `errchk` is ignored and we save the failure to the global var `errfound`

Reply:
Done